.gitignore

```diff
@@ -7,3 +7,4 @@ vendor
 devtools
 .env
 tmp
+cmd/neutree-cli/app/cmd/launch/manifests/*.tar
```

Comment:
looks like the tar files still included in this patch?

Reply:
uploading this file is just to fix go lint, maybe we can skip this go lint check

Reply:
the lint issue is not easy to handle. Can we keep it as it is?

Reply:
ok